src/index.rs

```diff
@@ -1729,6 +1729,29 @@ impl Index {
       .collect::<Result<Vec<InscriptionId>>>()
   }
 
+  pub(crate) fn get_runes_in_block(&self, block_height: u32) -> Result<Vec<RuneEntry>> {
+    let rtx = self.database.begin_read()?;
+
+    let rune_id_to_rune_entry = rtx.open_table(RUNE_ID_TO_RUNE_ENTRY)?;
+
+    let min_id = RuneId {
+      block: u64::from(block_height),
+      tx: 0,
+    };
+
+    let max_id = RuneId {
+      block: u64::from(block_height + 1),
+      tx: 0,
+    };
+
+    let runes = rune_id_to_rune_entry
+      .range(min_id.store()..max_id.store())?
```

Comment:
could you add a test, that checks whether the next blocks first rune is not included

Reply:
Will do it